Makefile

```diff
@@ -222,6 +222,13 @@ manager-aws-infrastructure: ## Build manager binary.
 clusterawsadm: ## Build clusterawsadm binary.
 	go build -ldflags "$(LDFLAGS)" -o $(BIN_DIR)/clusterawsadm ./cmd/clusterawsadm
 
+.PHONY: fetch-e2e-scripts
+fetch-e2e-scripts: ## Fetch start.sh and restart.sh and save it to /hack/tools
+	wget --output-document hack/tools/restart.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/restart.sh
+	wget --output-document hack/tools/start.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/start.sh
+	chmod +x hack/tools/start.sh
+	chmod +x hack/tools/restart.sh
+
```

Comment:
Shouldn't need to use phony for this actually. There's a much cleaner way of doing this that also ensures they would only be downloaded once if they're not there.